### PR TITLE
Provides contributors with a prompt to choose whether to overwrite, s…

### PR DESCRIPTION
**Pull request by bojeil-google**
…kip or abort the rules overwrite necessary for running integration tests.

### CODE REVIEW

**Comment by bojeil-google** [comment]
Will add a flag to force overwrite for automated scripts.

File: test/integration/index.js [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 var _ = require('lodash');
 var chalk = require('chalk');
+var readline = require('readline');
```
Comment (hiranya911) [inline test/integration/index.js]: Should we add this to `package.json`?

File: test/integration/index.js [hunk 2]
```diff
@@ -78,6 +79,58 @@ utils.assert(
...
+ * Prompts the developer whether the Database rules should be
+ * overwritten with the relevant rules for the integration tests.
+ * The developer has 3 options:
+ * yes/y to agree to the rules overwrite.
+ * skip to skip the overwrite (rules already manually configured) and continue
+ * with the tests.
+ * no/n or other to abort.
+ * @return {Promise} A promise that resolves when the rules change is processed.
+ */
+function promptForUpdateRules() {
```
Comment (hiranya911) [inline test/integration/index.js]: Lets also support an `--overwrite yes|no|skip` flag as discussed offline. For example:

`node test/integration --overwrite yes` should simply skip the prompt, and proceed to overwrite the rules.

File: test/integration/index.js (hunk 1 above)
Comment (bojeil-google) [inline test/integration/index.js]: I believe this module comes with node.js: https://nodejs.org/api/readline.html, similar to http module.

File: test/integration/index.js (hunk 2 above)
Comment (bojeil-google) [inline test/integration/index.js]: Yes, I will do so in my next commit. Thanks.

**Comment by bojeil-google** [comment]
You're welcome, @hiranya911 !

**Pull request merged**